data/schema/filters/abilities.cfg

```diff
@@ -0,0 +1,15 @@
+
+[tag]
+	name="$filter_abilities"
+	max=0
+	{SIMPLE_KEY id string_list}
+	{SIMPLE_KEY name string_list}
```

Comment:
Ability name is a translatable string, right? It's probably not a good idea to support filtering in a translatable string.

Reply:
Ok, i will remove name then, id sufficient like attribute without use name

Reply:
removed